auto_cpufreq/bin/auto_cpufreq.py

```diff
@@ -58,37 +48,20 @@ def config_info_dialog():
         run(["auto-cpufreq", "--help"])
         footer()
     else:
-        if daemon:
-            config_info_dialog()
-            root_check()
-            file_stats()
-            if os.getenv("PKG_MARKER") == "SNAP" and dcheck == "enabled":
-                gnome_power_detect_snap()
-                tlp_service_detect_snap()
-            elif os.getenv("PKG_MARKER") != "SNAP":
-                gnome_power_detect()
-                tlp_service_detect()
-            battery_setup()
-            conf.notifier.start()
-            while True:
-                try:
-                    footer()
-                    gov_check()
-                    cpufreqctl()
-                    distro_info()
-                    sysinfo()
-                    set_autofreq()
-                    countdown(2)
-                except KeyboardInterrupt: break
-            conf.notifier.stop()
-        elif monitor:
+        # set governor override unless None or invalid
+        if force is not None:
+            not_running_daemon_check()
+            root_check() # Calling root_check before set_override as it will require sudo access
+            set_override(force) # Calling set override, only if force has some values
+
```

Comment:
I fail to see how were all the lines on replaced with these new changes?  

Or was it moved to lines starting 113:

```
        elif daemon:
            config_info_dialog()
            root_check()
            file_stats()
            if IS_INSTALLED_WITH_SNAP and dcheck == "enabled":
                gnome_power_detect_snap()
                tlp_service_detect_snap()
            elif not IS_INSTALLED_WITH_SNAP:
                gnome_power_detect()
                tlp_service_detect()
            battery_setup()
            conf.notifier.start()
            while True:
                try:
                    footer()
                    gov_check()
                    cpufreqctl()
                    distro_info()
                    sysinfo()
                    set_autofreq()
                    countdown(2)
```

Reply:
The order has been changed to match the order of commands when you do `auto-cpufreq --help`.